manifests/charts/base/templates/validatingadmissionpolicy.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "stable-channel-policy.istio.io"
```

Comment:
the deployment of the stable channel policy vs the other policies can be done using the releaseChannel value set in base too. For eg, `{{- if eq .Values.base.releaseChannel "stable" }}`

Reply:
I didn't want to co-opt the release channel name YET, so I created an "experimental" namespace to signal users that the value name/path might change